src/Config/RuntimeConfigLoader.cs

```diff
@@ -133,7 +122,7 @@ public static bool TryParseConfig(string json,
             }
```

Comment:
We should put `Dictionary<string, string>? datasourceNameToConnectionString =  new Dictionary<string, string>();` in the function defination itself instead of putting null there and then checking and replacing it with new object here.